air-quality-ui/system_tests/ui/city_page.spec.ts

```diff
@@ -5,10 +5,6 @@ import {
   createMeasurementsCityPageResponseData,
 } from '../utils/mocked_api'
 
-test.use({
-  viewport: { width: 1920, height: 1080 },
-})
-
```

Comment:
Hey Ben, we may not need to delete this. From what I have seen the tests use the most recent test.use above them. So this test.use shouldn't affect your tests. Let me know if otherwise :)